init.el

```diff
@@ -93,6 +93,10 @@ Also contains along with versions and other config.")
 
        '( ;; Fixing weird quirks and poor defaults
          (:name better-defaults)
+         (:name plantuml-mode
+                :after (progn (setq plantuml-default-exec-mode 'jar)
+                              (add-to-list 'auto-mode-alist
+                                           '("\\.puml$" . plantuml-mode))))
```

Comment:
one minor change please: The recipes are ordered in alphabetical order - `better-defaults, company-mode` etc in the `setq` form. Can you please put `plantuml` in the correct place?